packages/angular/cli/utilities/config.ts

```diff
@@ -134,10 +168,13 @@ export function getWorkspaceRaw(
     }
   }
 
-  let content = '';
-  new NodeJsSyncHost().read(normalize(configPath))
-    .subscribe(data => content = virtualFs.fileBufferToString(data));
-
+  const data = readFileSync(configPath);
+  let start = 0;
+  if (data.length > 3 && data[0] === 0xef && data[1] === 0xbb && data[2] === 0xbf) {
+    // Remove BOM
+    start = 3;
+  }
+  const content = data.toString('utf-8', start);
```

Comment:
Maybe `data.toString('utf-8').replace(/^\uFEFF/, '')` to remove the BOM?

Reply:
Less efficient especially for large files due to the need to allocate two strings plus the regex cost.

Reply:
Fair enough